server/apps/archive/archive_composite.py

```diff
@@ -134,8 +134,11 @@ def update_link(self, package_id, assoc, delete=False):
         if not delete:
             two_way_links.append({'package': package_id})
 
+        # do update without changing etag/last_modified
         updates = self.get_item_update_data(item, two_way_links, delete)
-        get_resource_service(endpoint).patch(item_id, updates)
+        updates['_etag'] = item['_etag']
```

Comment:
I agree, I think a better solution is to use the Eve support for this: https://github.com/nicolaiarocci/eve/commit/d7e9bd024d9c1a5faad0e31b737d119cf3ee1fc7